utils/micro-controllers.stanza

```diff
@@ -186,6 +130,91 @@ public defn MicroController (properties:Tuple<KeyValue>, exist:Tuple<String>, so
   else :
     MicroController(jsons[0])
 
+;============================================================
+;======================= MCU to JITX ========================
+;============================================================
+
+defmethod to-jitx (mcu: MicroController) -> Instantiable :
+  pcb-component my-microcontroller :
+    manufacturer = manufacturer(mcu)
+    mpn = mpn(mcu)
+
+    to-jitx-pin-properties(pin-properties(mcu))
+    val bundle-table = to-jitx-bundles(bundles(mcu))
+    to-jitx-supports(supports(mcu), bundle-table)
+
+    assign-landpattern $
+      switch(mfg-package(mcu)) :
+        ; "SO8N" : SO8N
```

Comment:
If these are unnecessary delete the commented out lines. 

However some of these land patterns are encoded, so what is the purpose of the switch?